translate/src/modules/user/components/UserMenu.css

```diff
@@ -106,3 +106,72 @@
   margin: 5px 0;
   padding: 0;
 }
+
+/* Profile menu */
```

Comment:
This whole CSS file belongs to a User (Profile) Menu. Theme toggle would make more sense.